### PR TITLE
Add host endpoints to cli and use new node struct

### PR DESCRIPTION
**Pull request by alphadose**
Testing can be done when `netclient_refactor_latest` is merged into develop because it contains `models.ApiHost`

Merge this after `netclient_refactor_latest` is merged

### CODE REVIEW

File: cli/cmd/host/root.go [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+	"github.com/spf13/cobra"
+)
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "host",
+	Short: "Manage hosts",
+	Long:  `Manage hosts`,
+	// Run: func(cmd *cobra.Command, args []string) { },
```
Comment (Aceix) [inline cli/cmd/host/root.go:14]: remove?

File: cli/cmd/host/update.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	"encoding/json"
+	"log"
+	"os"
+
+	"github.com/gravitl/netmaker/cli/functions"
+	"github.com/spf13/cobra"
+)
+
+var hostUpdateCmd = &cobra.Command{
+	Use:   "update HostID /path/to/host_definition.json",
```
Comment (Aceix) [inline cli/cmd/host/update.go:13]: how do i know the fields of the json file?

Comment (alphadose) [reply cli/cmd/host/update.go:13]: will add a sample definition json once I get it

File: cli/cmd/host/root.go (hunk 1 above)
Comment (alphadose) [reply cli/cmd/host/root.go]: done

**Review by 0xdcarns — APPROVED** [review]

**Comment by alphadose** [comment]
Removed toggle flag and added flags for host update

**Review by 0xdcarns — APPROVED** [review]

**Review by mattkasun — APPROVED** [review]

**Review by Aceix — APPROVED** [review]

**Pull request merged**